<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>CopyDialog</class>
  <widget class="QDialog" name="CopyDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>303</width>
        <height>150</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Copy between cases</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <layout class="QVBoxLayout" >
          <item>
            <layout class="QHBoxLayout" >
              <item>
                <widget class="QLabel" name="FileNameLabel" >
                  <property name="text" >
                    <string>Copy file</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="FileName" >
                  <property name="frameShape" >
                    <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow" >
                    <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text" >
                    <string/>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <item>
                <widget class="QLabel" name="SourceCaseLabel" >
                  <property name="text" >
                    <string>From case</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="SourceCaseName" >
                  <property name="frameShape" >
                    <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow" >
                    <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="text" >
                    <string/>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <item>
                <widget class="QLabel" name="DestCaseLabel" >
                  <property name="text" >
                    <string>To case</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QComboBox" name="DestCaseCB" />
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <item>
                <spacer name="spacer3" >
                  <property name="sizeHint" >
                    <size>
                      <width>41</width>
                      <height>31</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="CopyBtn" >
                  <property name="text" >
                    <string>Copy</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="CancelBtn" >
                  <property name="text" >
                    <string>Cancel</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2" >
                  <property name="sizeHint" >
                    <size>
                      <width>61</width>
                      <height>31</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>CopyBtn</sender>
      <signal>clicked()</signal>
      <receiver>CopyDialog</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>CancelBtn</sender>
      <signal>clicked()</signal>
      <receiver>CopyDialog</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
